scripts/Dockerfile

```diff
@@ -1,24 +1,24 @@
 # Container for running Vivado on M1/M2 macs
 # though it should work equally on Intel macs
 FROM --platform=linux/amd64 ubuntu:22.04
-RUN apt update && apt upgrade -y
+RUN apt-get update && apt-get upgrade -y
 
 # install gui
-RUN apt install -y --no-install-recommends --allow-unauthenticated \
-dbus dbus-x11 x11-utils xorg alsa-utils mesa-utils net-tools \
-libgl1-mesa-dri gtk2-engines lxappearance fonts-droid-fallback sudo firefox \
-ubuntu-gnome-default-settings ca-certificates curl gnupg lxde arc-theme \
-gtk2-engines-murrine gtk2-engines-pixbuf gnome-themes-standard nano xterm
+RUN apt-get install -y --no-install-recommends \
+    dbus dbus-x11 x11-utils xorg alsa-utils mesa-utils net-tools \
+    libgl1-mesa-dri gtk2-engines lxappearance fonts-droid-fallback sudo firefox \
+    ubuntu-gnome-default-settings ca-certificates curl gnupg lxde arc-theme \
+    gtk2-engines-murrine gtk2-engines-pixbuf gnome-themes-standard nano xterm
 
 # install dependencies for Vivado
-RUN apt install -y --no-install-recommends --allow-unauthenticated \
```

Comment:
I did remove the `--allow-unauthenticated`. Was there a good reason to use that flag? It poses a security risk.

Reply:
No, I probably copy-pasted it without giving any thought to it...